.gitignore

```diff
@@ -1 +1,3 @@
 /internal/tests/testdata/graphql-js
+
+.idea/
```

Comment:
This looks like it's specific to your developer enviromment. It'd probably be best to add in your [global .gitignore](https://github.com/github/gitignore/tree/master/Global).